website/src/releases/2023-02/deployment-previews.mdx

```diff
@@ -0,0 +1,11 @@
+---
+date: 2023-02-24
+title: Custom Application deployment previews
+description: Introducing deployment previews for Custom Applications.
+type: announcement
+topics:
+  - Merchant Center
+  - Configuration
+---
+
+Custom Applications now lets you add deployment previews, where you can test new features before publishing your Custom Application to the production environment.
```

Comment:
Please add a link to the docs where we explain what the feature is and how to use it.

Reply:
Also, it might be good to have some sort of migration steps described.

Reply:
Sure. The release note is in WIP. @mmaltsev-ct will help me.
- I'll add the link to docs.
- Can you highlight a bit more on the migration steps required?

Reply:
Here is my suggestion for the release notes:

Reply:
Custom Applications now let you generate deployment previews, where you can test new features before publishing your application to the production environment. With the first version of this feature, it is possible to manually create static environments using the Merchant Center. A static URL, where the Custom Application will be deployed, can then be assigned to each of the environments.

You can read more about this feature in the new [Deployment previews](https://docs.commercetools.com/custom-applications/concepts/deployment-previews) chapter of our documentation.

As always, if you have questions or feedback, you can open a [GitHub Discussion](https://github.com/commercetools/merchant-center-application-kit/discussions) or a [GitHub Issue](https://github.com/commercetools/merchant-center-application-kit/issues).

Reply:
I think the migration steps should be better described in the docs themselves rather than in the release notes.

Reply:
Thanks, @mmaltsev-ct. I'll adapt your suggestion in the release note. Will you also review the release note on the MC PR https://github.com/commercetools/commercetools-docs/pull/2900?